http-server/server/server.go

```diff
@@ -11,17 +12,31 @@ import (
 )
 
 var pageAccessCount = 0
-var Param string
+var param string = ""
```

Comment:
Would be better to convert this to a struct:
```go
type server struct (
  param string
  pageAccessCount int
}
```

And then create a methods on this struct.